### ISSUE TITLE
kv.createReadStream({ live: true }) throwing on me

### ISSUE THREAD

**Issue by laduke** [substack/hyperkv#11]
Heya!
Not sure if I'm using it wrong, or something changed in level-live-stream or pull-stream? 

If it turn on `live`, something emits row `{ sync: true }`

which causes: 

```
TypeError: Cannot read property 'map' of undefined
    at DestroyableTransform.write [as _transform] (/Users/travis/src/hyperkv/index.js:169:24)
```

any tips?

Hrm, just wrapping the body of write() stuff in a `if (row.key)... else next()` seems to solve it and the tests still pass 😬

**Comment by noffle** [comment]
[pull-live](https://github.com/pull-stream/pull-live) emits a special `{ sync: true }` message when old history meets real-time. If you add something to ignore that message type (which it sounds like you've done) it won't bother you.

I'd be down with a patch that passed `opts.sync = false` into level-live-stream, which already passes its `opts` down to pull-live.

**Comment by laduke** [comment]
Thanks for responding! Let me give that a shot. 
I'm confused how people don't eventually run into the sync message?

**Comment by noffle** [comment]
It's possible you're the first person to use `createReadStream` with `live: true`!

**Comment by laduke** [comment]
closed! #12
